test/rest-api/binary.mocks.ts

```diff
@@ -0,0 +1,9 @@
+import { setupWorker, rest } from 'msw'
```

Comment:
Could you please rename this file to start with `body-`? We are getting more different body usage scenarios, it would be nice for them to be next to each other in the file tree.

```diff
- binary.mocks.ts
- binary.test.ts
+ body-binary.mocks.ts
+ body-binary.test.ts
```

Reply:
Okay, I renamed the files.